### PR TITLE
Add datetime to the Query Builder filter example

### PR DESCRIPTION
**Pull request by havatv**
Fixes: #2188

- [x] Backport to LTR documentation is required

### CODE REVIEW

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 1]
```diff
@@ -134,12 +134,19 @@ you to define a subset of the features in the layer using a SQL-like WHERE
...
-are of ``borough`` type in the project (see Figure_vector_querybuilder_ for
-such an example). The filter is made at the data provider (OGR, PostgreSQL,
-MSSQL...) level.
 
+You can use one or more layer attributes to define the filter in the `Query
+Builder`.
+An example of the use of more than one attribute is shown in
+Figure_vector_querybuilder_.
+In the example, the filter combines the attributes ``thetime``
+(``DateTime`` field: ``"thetime" > '2017-12-24 18:00:00```,
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]:
```suggestion
(``DateTime`` field: ``"thetime" > '2017-12-24 18:00:00'``,
```

**Comment by DelazJ** [comment]
Sorry but I don't really get the purpose of this change. Not sure of what it adds. It breaks translations and uses afaics dataset not available for others, meaning that it will probably get changed by the next person that would give a look to it and doesn't want to build adhoc layer structure.

**Comment by havatv** [comment]
@DelazJ, see #2188. The purpose is to show how to create filters using fields with different data types, and most importantly to use datetime in a filter, as suggested in #2188.

**Comment by DelazJ** [comment]
Unless I misread, the issuer is complaining about the undocumented requirement of casting datetime field in the query builder. Which is wrong since your example (that I suppose is a working one) does not cast such fields. So, either a bug fixed in QGIS meanwhile or an invalid issue (or an issue related to his underlining provider, which is ???)

Regarding dataset, training data is not used for user manual; it's for the training manual. The user manual has its own data, which is the Alaska dataset (mentioned in the getting started chapter, and probably in doc guidelines)

**Comment by havatv** [comment]
I have to fix more on this. If you are not aware of any elements of the Alaska dataset that contains a DateTime field, I will just continue to use a "purpose made" dataset.

**Comment by havatv** [comment]
The Shapefile format does not support DateTime, so I had to use GeoPackage. The syntax suggested in #2188 turned out to work for GeoPackage. So I hope it is closer to OK now.

**Pull request merged**